<launch>
  <arg name="fname" default="test_file"/>
  <arg name="csv_location" value="~/Desktop/CARIS/csvs"/>

  <!-- Tilde operator doesn't work here -->
  <arg name="plot_location" value="/home/leo/Desktop/CARIS/Plots"/>
  <param name ="/use_sim_time" value="true"/>
  
  <!-- <node name="read_csv_publisher" pkg="torque_controller" type="read_csv_publisher.py" /> -->
  <node name="csv_ang_vel_publisher" pkg="torque_controller" type="csv_ang_vel_publisher.py" />
  <node name="get_data_subscriber" pkg="torque_controller" type="get_data_subscriber.py" >
    <param name="csv_save_location" type="string" value="$(arg csv_location)" />
    <param name="plot_save_location" type="string" value="$(arg plot_location)" />
    <param name="file_name" type="string" value="$(arg fname)" />
  </node>
</launch>